src/System.Private.Interop/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -144,27 +144,27 @@ public static int SizeOf(Type t)
             if (t.TypeHandle.IsGenericType() || t.TypeHandle.IsGenericTypeDefinition())
                 throw new ArgumentException(SR.Argument_NeedNonGenericType, nameof(t));
 
-            return SizeOfHelper(t, true);
+            return SizeOfHelper(t);
         }
 
-        private static int SizeOfHelper(Type t, bool throwIfNotMarshalable)
```

Comment:
Why do you remove this paramter? it should  check whether the struct is marshallable or not. if not, throw exception.
Either add another paramter into RuntimeInteropData.Instance.TryGetStructUnsafeStructSize or add another method in RuntimeInteropData.Instance.

Reply:
This parameter was unused and always `true`.

Handling of invalid cases is very different between CoreRT and CoreCLR. I can open an issue on it if you believe that it is worth harmonizing, but I am not planning to do it in this PR.